compiler/one-cmds/one-prepare-venv

```diff
@@ -79,7 +78,7 @@ if [[ ! -z "$ONE_PREPVENV_TORCH_STABLE" ]]; then
 fi
 # TODO remove torch message
 echo "Torch from '${ONE_PREPVENV_TORCH_STABLE}' -> '${TORCH_STABLE_URL}'"
-${VENV_PYTHON} -m pip ${PIP_OPTIONS} install torch==1.11.0+cpu -f ${TORCH_STABLE_URL}
+${VENV_PYTHON} -m pip ${PIP_OPTIONS} install torch==1.13.1+cpu -f ${TORCH_STABLE_URL}
```

Comment:
Just curiosity. How torch version was decided? Is it a version near the TF release date?

Reply:
1.13.1 is last of 1.x version.
Next is 2.0, March 16, 2023.
I think upgrading to 2.x would be left to next step.

https://pypi.org/project/torch/#history